dnplab/dnpIO.py

```
@@ -0,0 +1,37 @@
+from . import dnpImport
+
+
+def import_data(path, data_type=None, *args, **kwargs):
+    """Import data from different spectrometer formats
+
+    Args:
+        path (str): Path to data directory or file
+        data_type (str): Type of data to import
+
+    Returns:
+        data (dnpData): Data object
+    """
+
+    if data_type == None:
+        data_type = autodetect(path)
+
+    if data_type == NotImplemented:
```

NotImplemented data type should raise an Exception?